magefiles/magefile.go

```diff
@@ -133,7 +133,8 @@ func (Local) CleanupGithubOrg() error {
 	}
 
 	// Get all repos
-	ghClient := github.NewGithubClient(githubToken, "redhat-appstudio-qe")
+	githubOrgName := utils.GetEnv("MY_GITHUB_ORG", "redhat-appstudio-qe")
```

Comment:
probably better to use the constant for the env var name instead of a string, similarly like [here](https://github.com/redhat-appstudio/e2e-tests/blob/28b9b0f3f7fd20c26c5812edd7630cc28e1b65c0/magefiles/magefile.go#L495)

otherwise lgtm